starlark/interp.go

```diff
@@ -503,7 +505,7 @@ loop:
 			dict, err2 := thread.Load(thread, module)
 			thread.beginProfSpan()
 			if err2 != nil {
-				err = fmt.Errorf("cannot load %s: %v", module, err2)
+				err = xerrors.Errorf("cannot load %s: %w", module, err2)
```

Comment:
I'm a little uneasy about this function, partly because I don't know exactly what it does, and partly because it adds an unnecessary dependency. IIUC, this code formats the error message and retains the underlying error, exposed through a Unwrap. It should be possible to construct an EvalError that does that (now that it has 'cause').

The other import of xerrors in the test can be eliminated by simply duplicating the relevant interface type declaration.

Can we replace this code with a new EvalError

Reply:
ya, i was bothered by the xerrors dependency too.

I added a new loadError that implements the same interface.

wrapping this as an EvalError isn't quite right, because (as you pointed out at the start of this review), EvalError has a backtrace and that backtrace would be wrong here (i.e., it makes the tests fail)